### ISSUE TITLE
First quest in "The Crown of Wills" quest chain unobtainable and incompletable if second quest is accepted first 

### ISSUE THREAD

**Issue by NeutyBooty** [Questie/Questie#2722]
## Description

"The Crown of Will" quest given by Sharlindra in Undercity starts The Crown of Will quest chain. This quest requires you to visit Melisara at Tarren Mill in the Hillsbrad foothills, who then gives you the remaining quests in the chain. However, if you go directly to Melisara first without speaking to Sharlindra beforehand, you are able to accept the second quest in the chain without needing to visit Sharlindra. This makes the first quest in the chain unobtainable and incompletable. 

However, Questie still lists the first quest as available and uncompleted in the "Quest by Zone" tab in the "My Journey" pane. "Advanced Search" for the quest also lists all of the other quests in the chain as completed once you've finished it except for the first one. 

Should Questie mark this first quest as complete once the second quest is accepted? Now there is a quest sitting in my Available Quests list for Undercity I'll never be able to clear out. 

Likewise, the options "Hidden by Questie" and "Complete" are grayed out in the Advanced Search tab of My Journey. Is there a way I can manually toggle either of these boxes, therefore circumventing the above issue?

## Wowhead link

First quest in chain:
https://classic.wowhead.com/quest=495/the-crown-of-will

Second quest in chain:
https://classic.wowhead.com/quest=518/the-crown-of-will

## Screenshots

![WoWScrnShot_050721_034716](https://user-images.githubusercontent.com/83809145/117418264-34b7b600-aee9-11eb-9f26-f1c5d3ed04b0.jpg)
![WoWScrnShot_050721_040645](https://user-images.githubusercontent.com/83809145/117418690-b4458500-aee9-11eb-85d1-31728718c667.jpg)

## Questie version

6.2.6 - Confirmed via minimap icon. Made sure Questie was fully upgraded before reporting this bug.

**Comment by BreakBB** [comment]
Hey @NeutyBooty thanks for this fantastic report. This has actually already been fixed for TBC, but I will move it to the Classic fixes.

For now you can simply "manually hide" the quest from your map by `Shift + Left click`ing on the icon. That won't change the state in your Journey though, but will be fixed with the next release 👍🏻

**Comment by NeutyBooty** [comment]
You guys rock! I'm a dev/DBA/SA for my career, so I've made it a habit to be _very thorough_ in my handovers when it comes to technical work.

Thanks for helping out! I know it's such a minor detail but I've become obsessed with clearing out the My Journey page for my character. You guys are the reason I decided to do a 100% completion run. Trying to clear all classic quests before TBC drop!

Will keep an eye out for the next release.
